src/driver/postgres/PostgresQueryRunner.ts

```diff
@@ -1917,6 +1922,14 @@ export class PostgresQueryRunner extends BaseQueryRunner implements QueryRunner
         return new Query(sql);
     }
 
+    /**
+     * Loads Postgres version.
+     */
+    protected async getVersion(): Promise<string> {
```

Comment:
Wouldn't it be better if `getVersion` moved to **PostgresDriver**, also cache the result as a property in the driver that could later be used directly in any subsequent scenarios? (ex: https://github.com/typeorm/typeorm/pull/7741)